core/crypto/sr25519_types.hpp

```diff
@@ -117,4 +116,25 @@ namespace kagome::crypto {
 
 }  // namespace kagome::crypto
 
+template<>
```

Comment:
Check this file for clang-format. My clang format suggests as follows:
```suggestion
template <>
```